pkg/nexus/nexus_test.go

```diff
@@ -0,0 +1,303 @@
+package nexus
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"testing"
+
+	piperhttp "github.com/SAP/jenkins-library/pkg/http"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUpload_AddArtifact(t *testing.T) {
```

Comment:
also minor but We never use '_' in our test names. I would stick to camel case

Reply:
The naming is what GoLand suggests (i.e. autocompletion of the test function name), but I have renamed them, since I also couldn't find any examples to the contrary.